### PR TITLE
Mnemonic mdr

### PR DESCRIPTION
**Pull request by konraduh**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
I small addition related to configuring whether incidents and remote cases should be closed during mirroring, as per customer request.

If this could be fast-tracked without demo, that would be appreciated.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by reutshal** [comment]
@konraduh Please fill the  [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**Comment by konraduh** [comment]
@reutshal this is already signed? This is an update to a published partner contribution.

**Comment by reutshal** [comment]
@konraduh We are asking to document every new contribution to keep track and also to make sure that changes are being applied only by the partners and not everyone :)

**Comment by konraduh** [comment]
My apologies, I misunderstood. Form filled.

File: Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml [hunk 1]
```diff
@@ -91,6 +105,106 @@ script:
...
+    arguments:
+    - name: case_id
+      required: true
+      description: ID of Argus case
+    - name: file_id
+      required: true
+      description: ID of attachement
+    outputs:
+    - contextPath: Argus.Attachment.responseCode
+      description: API response metadata, response code of this request
```
Comment (reutshal) [inline Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml:118]:
```suggestion
      description: The response code of this request.
```

I Don't think we need the `API response metadata` prefix, we want the description to be as short as possible.
True for all outputs descriptions.
You can read more about outputs description here: https://xsoar.pan.dev/docs/documentation/documentation_tips#outputs

File: Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml [hunk 2]
```diff
@@ -91,6 +105,106 @@ script:
...
+      description: API response metadata, the offset into the result-set of this query
+      type: Number
+    - contextPath: Argus.Attachment.count
+      description: API response metadata, total number of results this query has
+      type: Number
+    - contextPath: Argus.Attachment.size
+      description: API response metadata, the number of results returned in this request
+      type: Number
+    - contextPath: Argus.Attachment.messages.message
+      description: Attachment Messages Message
```
Comment (reutshal) [inline Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml:133]:
```suggestion
      description: Attachment Messages Message
```

Can you provide a better description here? what is messages message?

**Review by reutshal — CHANGES_REQUESTED** [review]
Overall looks good! Please see my few notes

**Comment by reutshal** [comment]
@sangsoojun88 Can you please take a look? Thanks

File: Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml (hunk 1 above)
Comment (konraduh) [reply Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml:118]: This has been approved before, and I do not have the time to change this now. I would appreciate if we could focus on the changes in this PR, unless there are critical issues.

File: Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml (hunk 2 above)
Comment (konraduh) [reply Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml:133]: This has been approved before, and I do not have the time to change this now. I would appreciate if we could focus on the changes in this PR, unless there are critical issues.

**Review by reutshal — APPROVED** [review]

**Comment by konraduh** [comment]
Thank you! :)

**Pull request merged**